ir/instr.cpp

```diff
@@ -2309,7 +2225,7 @@ StateValue Return::toSMT(State &s) const {
 
   auto &attrs = s.getFn().getFnAttrs();
   if (attrs.poisonImpliesUB())
-    retval = s.getAndAddPoisonUB(*val);
+    retval = s.getAndAddPoisonUB(*val, attrs.undefImpliesUB());
```

Comment:
cant we add an assert to attributes to check that undefImpliesUB() implies poisonImpliesUB()?  Otherwise this code makes me nervous.

Reply:
Added the assertion at undefImpliesUB